scripts/notify-slack.sh

```diff
@@ -0,0 +1,53 @@
+#!/bin/bash
+
+curl \
+  -X POST \
+  -H "Content-Type: application/json" \
+  --data '
+
+  {
+    "blocks": [
+      {
+        "type": "context",
+        "elements": [
+          {
+            "type": "mrkdwn",
+            "text": ":rocket: *Preview Deployed*"
```

Comment:
Instead of saying **Preview** - using the branch name might be more clear as to what is being "deployed". if we do this, we could remove the `context` section of the message.